crate_universe/repositories.bzl

```diff
@@ -12,7 +12,9 @@ load("//crate_universe/tools/cross_installer:cross_installer_deps.bzl", "cross_i
 
 USE_CRATES_REPOSITORY = False
 
-_REPOSITORY_NAME = "crate_universe_crate_index"
+# Short for 'crate universe index'. Keep this short to reduce the risk to
+# bump into absolute path length issues on Windows.
```

Comment:
Can you update the comment to reference the tracking issue (https://github.com/bazelbuild/rules_rust/issues/1120) that would solve for this?
```suggestion
# bump into absolute path length issues on Windows. See:
# https://github.com/bazelbuild/rules_rust/issues/1120
```